frontend/src/containers/pages/PlayScenarioPage/PlayScenarioPage.jsx

```diff
@@ -39,37 +39,33 @@ export default function PlayScenarioPage() {
   const { scenarioId, sceneId } = useParams();
   const history = useHistory();
   const styles = useStyles();
-
   const [previous, setPrevious] = useState(null);
-  const [error, setError] = useState(null);
 
-  // TODO: move these somewhere else ?
-  if (loading) return <LoadingPage text="Loading Scene..." />;
-  if (authError) return <></>;
+  const setError = useCallback((error) => {
+    if (!error) return;
+    if (error.status === 409) {
+      history.push(`/play/${scenarioId}/desync`);
+    } else {
+      history.push(`/play/${scenarioId}/error`);
+    }
+  }, []);
```

Comment:
This function won't change if `scenarioId` changes, is this intended? Otherwise should add to the dependency array

Reply:
scenarioId changing is not something that needs to be considered here because it would cause the component to fully remount. In the normal logic there is no way to change the scenarioId while this component is rendered.

Reply:
was there any specific reason to use `useCallback` btw?

Reply:
not really, I was going to put it outside the component initially (for clarity). Good practice i guess?

Reply:
yeah ill just remove the `useCallback` then, serves no purpose here